lib/Doctrine/DBAL/Schema/Index.php

```diff
@@ -15,6 +15,8 @@
 
 class Index extends AbstractAsset implements Constraint
 {
+    public const FLAG_SPATIAL = 'spatial';
```

Comment:
The Public scope isn't needed for this constant. There's currently only one place in code where this constant is in use. We probably don't need `'spatial'` as a constant for now.

Reply:
Agree. Please see https://github.com/doctrine/dbal/pull/3970#discussion_r449196691.